pex/third_party/__init__.py

```diff
@@ -338,24 +339,31 @@ def isolated():
   global _ISOLATED
   if _ISOLATED is None:
     from pex import vendor
-    from pex.common import atomic_directory, safe_copy
+    from pex.common import atomic_directory
     from pex.util import CacheHelper
     from pex.variables import ENV
+    from pex.third_party.pkg_resources import resource_isdir, resource_listdir, resource_stream
+
+    module = 'pex'
+    def recursive_copy(srcdir, dstdir):
+      os.mkdir(dstdir)
+      for entry_name in resource_listdir(module, srcdir):
+        # NB: Resource path components are always separated by /, on all systems.
+        src_entry = '%s/%s' % (srcdir, entry_name) if srcdir else entry_name
```

Comment:
Nit: this can use `.format()`. Pex uses both styles, but I think John has recently been converting things to `.format()` because it's easier to read.